src/service/DatasetEditManager.ts

```diff
@@ -55,14 +57,22 @@ export class DatasetEditManager {
     ) {
         subscriptions.push(
             vscode.commands.registerCommand(
-                "zosexplorer.editMember",
+                "zosexplorer.edit",
                 async (arg) => {
+                    let passedMember: Member = { name: ""};
+                    const DSO: string = arg.dataset.dataSetOrganization;
+                    if (DSO === DSOrg.PO) {
+                        passedMember = arg.member;
+                    } else {
+                        passedMember.name = arg.dataset.name;
```

Comment:
Assignment dataset name to member seems wrong. Please make this solution more obvious.

Reply:
Slightly modified the above suggested review: please let me know.